Build error on Mac

Not sure you guys have noticed this or I'm doing something wrong.

Latest `master`: `2c5359e284cd22a7ab741f149a130dabd85036c6`

```
➜  tidb git:(master) make
go get github.com/tools/godep
go get github.com/pingcap/go-hbase
go get github.com/pingcap/go-themis
go get github.com/ngaut/tso/client
go get github.com/qiuyesuifeng/goyacc
go get github.com/qiuyesuifeng/golex
$(which goyacc) -o /dev/null -xegen temp_parser_file parser/parser.y
Parse table entries: 296474 of 772007, x 16 bits == 592948 bytes
$(which goyacc) -o parser/parser.go -xe temp_parser_file parser/parser.y 2>&1 | egrep "(shift|reduce)/reduce" | awk '{print} END {if (NR > 0) {print "Find conflict in parser.y. Please check y.output for more information."; system("rm -f temp_parser_file"); exit 1;}}'
rm -f temp_parser_file
rm -f y.output
$(which golex) -o parser/scanner.go parser/scanner.l
$(which godep) go build
# github.com/pingcap/tidb/store/hbase
store/hbase/kv.go:90: cannot use t (type *themis.Txn) as type themis.Txn in argument to newHbaseTxn
store/hbase/kv.go:100: cannot use t (type *themis.Txn) as type themis.Txn in argument to newHbaseSnapshot
store/hbase/snapshot.go:66: s.txn.Gets undefined (type themis.Txn has no field or method Gets)
store/hbase/snapshot.go:154: cannot convert nil to type themis.Txn
store/hbase/snapshot.go:156: cannot use nil as type themis.Txn in assignment
godep: go exit status 2
make: *** [build] Error 1
```

Have received the email.
In the file, i see 
```
*** Test killed with quit: ran too long (10m0s).
FAIL    github.com/pingcap/tidb/store/localstore    600.122s

Makefile:111: recipe for target 'gotest' failed
```
In our store/localstor/isolation_test.go, there are two time-consuming tests `TestInc` and `TestMultiInc`.
Maybe we should move these tests out of tidb or reduce the pressure as @ngaut  metioned.
Now u can simply use `make godep && make parser && make build && make check[optional]` instead of `make` to skip the `make test`.

@Unknwon Thanks again.

--- a/store/localstore/compactor.go
+++ b/store/localstore/compactor.go
@@ -77,7 +77,9 @@ func (gc *localstoreCompactor) getAllVersions(k kv.Key) ([]kv.EncodedKey, error)
 	for it.Next() {
 		if kv.EncodedKey(it.Key()).Cmp(endKey) < 0 {
 			ret = append(ret, bytes.CloneBytes(kv.EncodedKey(it.Key())))
+			continue
 		}
+		break
 	}
 	return ret, nil
 }

